Harvest.Web/ClientApp/src/Requests/Crops.tsx

```diff
@@ -40,8 +43,29 @@ export const Crops = (props: Props) => {
     }
   };
 
+  const onSearch = async (query?: string) => {
+    setIsSearchLoading(true);
+
+    const response = await fetch(
+      `/Crop/Search?type=${props.cropType}${(query && `&query=${query}`) || ""}`
```

Comment:
I'm ok with leaving the query param in the server method but I would rather you just remove it from here to make it more readable.